### DISCUSSION TITLE
Allowing Style Overrides for Component Library

### DISCUSSION THREAD

**Discussion by helgardferreira** [unocss/unocss#3017] [Q&A]
I've created a simple POC svelte component library here: https://github.com/frappydev/dui using uno and I plan on making it possible for other svelte / sveltekit projects to import this library and use it. So far it's working but I'm struggling with style overrides.

To be specific, the issue I'm having is if a style (let's say background color) is specified on a component in the component library (dui in this case) and that component is then imported into a svelte project - how can I go about overriding this style in a project that also uses uno?

In summary - what's the best way to create a component library using unocss and then consume that library in another unocss project while allowing the consumer project to override styles?

**Comment by siduck** [comment]
i'd use the attributify plugin and merge props
